tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/api/GdcDetailsResource.java

```diff
@@ -91,7 +91,8 @@ public ResponseEntity<GdcDetailsDTO> updateGdcDetails(
       @RequestBody @Validated(Update.class) GdcDetailsDTO gdcDetailsDTO)
       throws URISyntaxException, MethodArgumentNotValidException {
     log.debug("REST request to update GdcDetails : {}", gdcDetailsDTO);
-    gdcDetailsValidator.validate(gdcDetailsDTO);
+    GdcDetailsDTO originalDto = gdcDetailsService.findOne(gdcDetailsDTO.getId());
+    gdcDetailsValidator.validate(gdcDetailsDTO, originalDto, Update.class);
```

Comment:
If refactoring, this could be placed after the id check like `ContactDetailsResource.updateContactDetails()`.